src/embed_builder.ts

```diff
@@ -76,7 +76,7 @@ export class EmbedBuilder<T> {
     const embedDomain = window.location.origin
     this._params = {
       embed_domain: embedDomain,
-      sdk: '1'
+      sdk: '2'
```

Comment:
Not for this PR, but it would be nice for this to be a symbolic constant used everywhere.